lib/solaar/upower.py

```diff
@@ -58,8 +58,8 @@ def watch(on_resume_callback=None, on_suspend_callback=None):
 
 	_UPOWER_BUS = 'org.freedesktop.UPower'
 	_UPOWER_INTERFACE = 'org.freedesktop.UPower'
-	_LOGIND_BUS = 'org.freedesktop.login1.Manager'
-	_LOGIND_INTERFACE = 'org.freedesktop.login1'
+	_LOGIND_BUS = 'org.freedesktop.login1'
+	_LOGIND_INTERFACE = 'org.freedesktop.login1.Manager'
```

Comment:
Why switching the lines?

Reply:
The values were switched.  The call that used the values had them in the wrong order so that cancelled out the wrong values.  I fixed the values and used named arguments like the other calls.  (I had copied the call from somewhere else and it didn't use named arguments.)

Reply:
Ah, sorry.